cmd/scheduler/app/options/options.go

```diff
@@ -78,6 +80,7 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 		"Enable PriorityClass to provide the capacity of preemption at pod group level; to disable it, set it false")
 	fs.Float32Var(&s.KubeAPIQPS, "kube-api-qps", defaultQPS, "QPS to use while talking with kubernetes apiserver")
 	fs.IntVar(&s.KubeAPIBurst, "kube-api-burst", defaultBurst, "Burst to use while talking with kubernetes apiserver")
+	fs.Float64Var(&s.EnqueueThreshold, "enqueue-threshold", defaultEnqueueThreshold, "Enqueue action threshold")
```

Comment:
This is not clear, @k82cn professional suggestions?

Reply:
We need to document this at least.

Reply:
Yes, what it means. So users can set this accordingly.